### DISCUSSION TITLE
Using cyclonedx-editor-validator to amend SBOM with license data generated by Conan

### DISCUSSION THREAD

**Discussion by cedricwritescode** [Festo-se/cyclonedx-editor-validator#166] [Q&A]
According to your specification, in order to amend an SBOM with license information the license text files must be named after their licenses.
Does this behaviour implicitly support Conan with its `_collected_license_files` folder (as in #165)?
If this is the case, I think it would be best to update the readme for cyclonedx-editor-validator.

**Comment by italvi** [comment]
Should be resolved by #160
